draft-ietf-ppm-dap.md

```diff
@@ -2046,23 +2087,27 @@ variant {
 } PrepareResp;
 ~~~
 
-Otherwise the Helper responds with
+Otherwise `state` has type `Continued` or `FinishedWithOutbound` and
+there is at least one more outbound message to process. (State `Finished` is
+not reachable at this point.) The Helper responds with
 
 ~~~ tls-presentation
 variant {
   ReportID report_id;
   PrepareRespType prepare_resp_type = continue;
-  opaque payload<0..2^32-1> = outbound;
+  opaque payload<0..2^32-1> = state.outbound;
 } PrepareResp;
 ~~~
 
-If `state == Continued(prep_state)`, then the Helper stores `state` to
-prepare for the next continuation step ({{aggregation-helper-continuation}}).
+If `state` has type `Continued`, then the Helper stores `state`
+for use in the first continuation step in
+({{aggregation-helper-continuation}}).
 
-If `state == Finished(out_share)`, then the Helper commits to `out_share` as
-described in {{batch-buckets}}. If commitment fails with some `ReportError`
-(e.g., the report was replayed or its batch bucket was collected), then instead
-of the `continue` response above, the Helper responds with
+Else if `state` has type `FinishedWithOutbound`, then the Helper commits to
+`state.out_share` as described in {{batch-buckets}}. If commitment fails with
+some `ReportError` (e.g., the report was replayed or its batch bucket was
+collected), then instead of the `continue` response above, the Helper responds
+with
```

Comment:
@tgeoghegan in a later PR it would be good to avoid

> then instead of the [...] response above, the Helper responds with

Ideally the text matches chronologically what we would do in the code.